tests/wycheproof_eddsa.rs

```diff
@@ -1,18 +1,18 @@
 #[cfg(test)]
-mod wycheproof_tests {
+mod wycheproof {
 
-    use wycheproof_gen::test_eddsa;
+    use wycheproof_gen::test_wycheproof;
 
-    use wycheproof::eddsa::*;
+    use wycheproof::wycheproof::*;
 
     use core::convert::TryFrom;
     use salty::{PublicKey, Signature};
     use salty::constants::{PUBLICKEY_SERIALIZED_LENGTH, SIGNATURE_SERIALIZED_LENGTH};
 
-    #[test_eddsa("tests/eddsa_test.json", "eddsa_verify_schema.json")]
-    fn eddsa_testcase (tg: &EddsaTestGroup, tc: &SignatureTestVector) {
+    #[test_wycheproof("tests/eddsa_test.json", "eddsa_verify_schema.json")]
+    fn eddsa_tc (key: &Key, tc: &SignatureTestVector) {
```

Comment:
Same here, nothing wrong with `test_case` instead of `tc`, when reading the code it's one less mental step to understand.

Reply:
Done.